src/storage/index.ts

```diff
@@ -0,0 +1,79 @@
+import express = require('express')
+import { server } from 'decentraland-server'
+
+import * as multer from 'multer'
+import * as multerS3 from 'multer-s3'
+
+import {
+  bucketName,
+  checkFile,
+  parseFileBody,
+  readFile,
+  s3,
+  uploadFile
+} from '../S3'
+
+const ENTRY_FILENAME = 'entry.json'
+const MAX_FILE_SIZE = 3000000
```

Comment:
Let's make it 5mb, this will apply to the JSON as well

Reply:
The limit for the JSON is set at a different level:
`this.app.use(bodyParser.json({ limit: '4mb' }))` 

I will change that one to `5mb`.